AIPscan/Aggregator/views.py

```diff
@@ -1,23 +1,22 @@
 # -*- coding: utf-8 -*-
 
-from datetime import datetime
 import os
 import shutil
+from datetime import datetime
 
 from celery.result import AsyncResult
-from flask import Blueprint, render_template, redirect, request, flash, url_for, jsonify
+from flask import Blueprint, flash, jsonify, redirect, render_template, request, url_for
 
 from AIPscan import db
-from AIPscan.Aggregator.task_helpers import get_packages_directory
-from AIPscan.Aggregator.forms import StorageServiceForm
 from AIPscan.Aggregator import tasks
+from AIPscan.Aggregator.forms import StorageServiceForm
+from AIPscan.Aggregator.task_helpers import get_packages_directory
 from AIPscan.extensions import celery
-from AIPscan.models import (
+from AIPscan.models import (  # Custom celery Models.
```

Comment:
This comment ended up in a funny place. Perhaps it could be removed?

Reply:
Oh weird, that may have started in a weird place too? No idea why! Correcting it seems to have put it back in a sensible place. Great catch!

**Before**

```python
from AIPscan.models import (	
    FetchJob,	# Custom celery Models.
    StorageService,	from AIPscan.models import FetchJob, StorageService, get_mets_tasks, package_tasks
    # Custom celery Models.	
    package_tasks,	
    get_mets_tasks,	
)
```

**After**

```python
# Custom celery Models.
from AIPscan.models import FetchJob, StorageService, get_mets_tasks, package_tasks
```